app/src/components/Details.tsx

```diff
@@ -17,7 +18,17 @@ export function Details() {
         &times;
       </CloseButton>
       <BoxTitle title={clickedBlob.name}>{clickedBlob.name}</BoxTitle>
+      <>
+        <strong>Path:</strong> {clickedBlob.path}
```

Comment:
![image](https://user-images.githubusercontent.com/1959615/157053082-bedf486a-baee-466e-86c8-273ea98b31c5.png)
This overflows if path is too long. Add ellipsis or text wrapping.